src/index.d.ts

```diff
@@ -94,7 +96,9 @@ export interface FormSpyProps extends RenderableProps<FormSpyRenderProps> {
 }
 
 export var Field: React.ComponentType<FieldProps>
-export var Form: React.ComponentType<FormProps>
+export class Form<FormData = object> extends React.Component<
```

Comment:
`Form` is actually implemented as a function, not as a class, so this type could be misleading couldn't it?

It would suggest I can create instances a la `const form = new Form()`?

Reply:
I've been doing this if it helps. 
```
<FormData = object>(
  props: FormProps<FormData>,
) => React.ReactElement<FormProps<FormData>>
```

Reply:
[It's `React.FC` now](https://github.com/final-form/react-final-form/blob/master/typescript/index.d.ts#L102).